src/pages/docs/ai-transport/features/sessions-identity/online-status.mdx

```diff
@@ -0,0 +1,279 @@
+  authUrl: "/api/auth/token"
+});
+
+const channel = ably.channels.get("{{RANDOM_CHANNEL_NAME}}");
+
+// Enter presence with metadata about the user's device
+await channel.presence.enter({
+  device: "mobile",
+  platform: "ios"
+});
+```
+</Code>
+
+<Code>
+```javascript
+// Agent code
+import * as Ably from "ably";
+
+const ably = new Ably.Realtime({
+  key: "{{API_KEY}}",
+  clientId: "weather-agent"
+});
+
+const channel = ably.channels.get("{{RANDOM_CHANNEL_NAME}}");
+
+// Enter presence with metadata about the agent
+await channel.presence.enter({
+  model: "gpt-4"
+});
+```
+</Code>
+
+### Going online from multiple devices <a id="multiple-devices"/>
+
+A single user can be present on a channel from multiple devices simultaneously. Ably tracks each connection separately using a unique [`connectionId`](/docs/connect#connection-ids), while maintaining the same [`clientId`](/docs/auth/identified-clients#assign) across all connections.
+
+When a user connects from multiple devices, each device enters presence independently. All connections share the same `clientId` but have different `connectionId` values:
+
+<Code>
+```javascript
+// Client code (device 1: desktop browser)
+import * as Ably from "ably";
+
+const ably = new Ably.Realtime({
+  authUrl: "/api/auth/token"
+});
+
+const channel = ably.channels.get("{{RANDOM_CHANNEL_NAME}}");
+await channel.presence.enter({ device: "desktop" });
+```
+</Code>
+
+<Code>
+```javascript
+// Client code (device 2: mobile app)
+import * as Ably from "ably";
+
+const ably = new Ably.Realtime({
+  authUrl: "/api/auth/token"
+});
+
+const channel = ably.channels.get("{{RANDOM_CHANNEL_NAME}}");
+await channel.presence.enter({ device: "mobile" });
+```
+</Code>
+
+When either device leaves or disconnects, the other device remains in the presence set.
+
+## Going offline <a id="going-offline"/>
+
+### Explicitly going offline <a id="leaving"/>
+
+Use the [`leave()`](/docs/presence-occupancy/presence#leave) method when a user or agent wants to mark themselves as offline. This immediately notifies presence subscribers on the channel and removes the entry from the presence set, even if they remain connected to Ably.
+
+<Aside data-type="note">
+Clients require the [`presence`](/docs/auth/capabilities) capability on the channel to leave presence.
+</Aside>
+
+<Code>
+```javascript
+// Client code
+import * as Ably from "ably";
+
+const ably = new Ably.Realtime({
+  authUrl: "/api/auth/token"
+});
+
+const channel = ably.channels.get("{{RANDOM_CHANNEL_NAME}}");
+
+// Leave presence when the user marks themselves offline
+await channel.presence.leave();
+```
+</Code>
+
+<Code>
+```javascript
+// Agent code
+import * as Ably from "ably";
+
+const ably = new Ably.Realtime({
+  key: "{{API_KEY}}",
+  clientId: "weather-agent"
+});
+
+const channel = ably.channels.get("{{RANDOM_CHANNEL_NAME}}");
+
+// Leave presence when the agent shuts down
+await channel.presence.leave();
+```
+</Code>
+
```

Comment:
Is it worth mentioning that a `close()` will cause an immediate `leave` event, for the cases that an agent or user is intentionally ending the conversation.
Also, an explicit `leave()` can contain a payload, and this might be useful for indicating a reason for leaving.